lib/tpl/main.php

```diff
@@ -14,7 +14,7 @@
 
         <title>
             <?php if ($tube) echo $tube . ' - ' ?>
-            <?php echo $serverLabel ? $serverLabel : 'All servers' ?> -
+            <?php echo $serverLabel ?? 'All servers'; ?> -

```

Comment:
This will not work on older PHP versions - a more compatible fix is below;

```suggestion
            <?php echo isset($serverLabel) ? $serverLabel : 'All servers' ?> -
```

Reply:
Its been available since PHP7, should increase minimum required version.

Reply:
The software supports PHP 5 - the change I suggested is in keeping with the rest of the file, see https://github.com/ptrofimov/beanstalk_console/blob/bc3386f4d2ae9e08d0d5333186c32e636f38d08b/lib/tpl/main.php#L28